src/AbstractFixer.php

```diff
@@ -65,6 +66,15 @@ public function __construct()
         }
     }
 
+    public function getExtraInfoFixers(): array
+    {
+        return [
+            'helpUri' => $this->getHelpUri(),
+            'definition' => $this->getDefinition(),
+            'risky' => $this->isRisky(),
```

Comment:
huh? no proxy methods please
we already have `isRisky()` and `getDefinition()`, no need for method that is duplicating this knowledge

Reply:
If you don't want to have external SARIF output format to your project I understand. Otherwise, consider that, as `helpUri` the extra info that missed from `ReportSummary` class, to generate full (validated) SARIF results which identify complety all rules/fixers applied !

I know that is not in context of this PR, but as we try to add an additional info (`helpUri`) here I took advange of the situation.

Reply:
I believe we should focus on minimal changes required for achieving clickable URLs in the output - nothing more, nothing less. That would make this PR much easier to process and finish. Then, it could be extender or/and refactored to achieve SARIF reporting - for now I don't even have the context what it is, what it needs and I don't want to dig into that at this point. Having said that, I agree with @keradus' [comment](#pullrequestreview-2051337940), and I would prefer first option from [this comment](#issuecomment-2106256357). Let's don't model stuff for things that will or won't land later.

That's why I don't like `FixerReportInterface` because we don't need to focus here on reporting (clickable URL can be also added to `describe` CLI command), I would prefer something more atomic, focused on docs URI. `ProvidesDocumentationUriInterface` or something like that, that would allow adding only this particular feature to fixers.

Reply:
FYI: If you wan't to know what is SARIF, consider to take time to read this page https://docs.github.com/en/code-security/code-scanning/integrating-with-code-scanning/sarif-support-for-code-scanning.
It's a good intro !